services/blockchain-service/implementations/node-blockchain-service.js

```diff
@@ -33,15 +33,33 @@ class NodeBlockchainService extends BlockchainServiceBase {
 
     async executeContractFunction(contractName, functionName, args, blockchain) {
         const web3Instance = await this.getWeb3Instance(blockchain);
+        let result;
```

Comment:
This implementation is similar to implementation on the [OT-node](https://github.com/OriginTrail/ot-node/blob/v6/develop/src/modules/blockchain/implementation/web3-service.js#L345).